spec/index.bs

```diff
@@ -596,24 +596,36 @@ Issue(gpuweb/gpuweb#296): Consider merging all read-only usages.
 Textures may consist of separate [=mipmap levels=] and [=array layers=],
 which can be used differently at any given time.
 Each such <dfn dfn>subresource</dfn> is uniquely identified by a
-[=texture=], [=mipmap level=], and
-(for {{GPUTextureDimension/2d}} textures only) [=array layer=].
+[=texture=], [=mipmap level=],
+(for {{GPUTextureDimension/2d}} textures only) [=array layer=],
+and [=aspect=].
 
-The **main usage rule** is that any [=subresource=]
-at any given time can only be in either:
+<dfn dfn>atom resource</dfn> means a whole buffer or a single subresource of a texture.
+
+The **main usage rule** is that any [=atom resource=]
+at any given time within [=usage scope=] can only be in either:
   - a combination of [=read-only usage=]s
   - a single [=mutating usage=]
 
+The only exception is that a combination of writeonly-storage-texture usages, or a combination
+of storage-buffer usages upon the same [#atom resource#] in render pass is allowed.
+
+Note that all invisible usages and overwritten usages should be counted, for example:
```

Comment:
is it meant to be normative? Otherwise, it could start with `Note:` to get the special syntax

Reply:
Done. Thanks for your suggestion!